py3status/py3.py

```diff
@@ -1157,6 +1160,11 @@ def request(self, url, params=None, data=None, headers=None,
         if headers is None:
             headers = {}
 
+        if 'User-Agent' not in headers:
+            headers['User-Agent'] = 'py3status/{} {}'.format(
+                version, self._uid
```

Comment:
Is UUID the issue here or just lack of `User-Agent`? Otherwise, we could do `py3status version 3.11 (python 3.7.0)`.

Reply:
UUID is

Reply:
Okay. This works for me and will work for users 100%. It looks good.